_config.yml

```diff
@@ -1,5 +1,4 @@
 name: Before you ship
-baseurl: /before-you-ship
```

Comment:
Confirmed locally that this change [broke the build](https://travis-ci.org/18F/before-you-ship/builds/105597827).